2.1.1 fails to install on windows / ELIFECYCLE

2.1.1 is failing to install on Windows (appveyor CI)
```
$ node -v
4.2.4
$ npm -v
2.14.12
```

```
npm install -g
C:\Users\appveyor\AppData\Roaming\npm\phantomjs -> C:\Users\appveyor\AppData\Roaming\npm\node_modules\phantomjs\bin\phantomjs
 
> phantomjs@2.1.1 install C:\Users\appveyor\AppData\Roaming\npm\node_modules\phantomjs
> node install.js
 
Looks like an `npm install -g` on windows; unable to check for already installed version.
Downloading https://bitbucket.org/ariya/phantomjs/downloads/phantomjs-2.1.1-windows.zip
Saving to C:\Users\appveyor\AppData\Local\Temp\1\phantomjs\phantomjs-2.1.1-windows.zip
Receiving...
 
Received 17767K total.
Extracting zip contents
Removing C:\Users\appveyor\AppData\Roaming\npm\node_modules\phantomjs\lib\phantom
Copying extracted folder C:\Users\appveyor\AppData\Local\Temp\1\phantomjs\phantomjs-2.1.1-windows.zip-extract-1453759798332\phantomjs-2.1.1-windows -> C:\Users\appveyor\AppData\Roaming\npm\node_modules\phantomjs\lib\phantom
chmod failed: phantomjs was not successfully copied to C:\Users\appveyor\AppData\Roaming\npm\node_modules\phantomjs\lib\phantom\phantomjs.exe
npm ERR! Windows_NT 6.3.9600
npm ERR! argv "C:\\Program Files (x86)\\nodejs\\node.exe" "C:\\Program Files (x86)\\nodejs\\node_modules\\npm\\bin\\npm-cli.js" "install" "-g" "phantomjs"
npm ERR! node v4.2.4
npm ERR! npm  v2.14.12
npm ERR! code ELIFECYCLE
 
npm ERR! phantomjs@2.1.1 install: `node install.js`
```

Full run:
https://ci.appveyor.com/project/johanneswuerbach/testem/build/748/job/fd1om6g152old123

I'm seeing this on OS X and Linux:

    npm ERR! Darwin 14.5.0
    npm ERR! argv "/Users/xxx/.nvm/versions/node/v5.2.0/bin/node" "/Users/ronaldjadams/.nvm/versions/node/v5.2.0/bin/npm" "install"
    npm ERR! node v5.2.0
    npm ERR! npm  v3.3.12
    npm ERR! code ELIFECYCLE
    npm ERR! phantomjs@2.1.1 install: `node install.js`
    npm ERR! Exit status 1
    npm ERR!
    npm ERR! Failed at the phantomjs@2.1.1 install script 'node install.js'.
    npm ERR! Make sure you have the latest version of node.js and npm installed.
    npm ERR! If you do, this is most likely a problem with the phantomjs package,
    npm ERR! not with npm itself.
    npm ERR! Tell the author that this fails on your system:
    npm ERR!     node install.js
    npm ERR! You can get their info via:
    npm ERR!     npm owner ls phantomjs
    npm ERR! There is likely additional logging output above.

--- a/install.js
+++ b/install.js
@@ -61,7 +61,7 @@ kew.resolve(true)
   })
   .then(function () {
     var location = getTargetPlatform() === 'win32' ?
-        path.join(pkgPath, 'phantomjs.exe') :
+        path.join(pkgPath, 'bin', 'phantomjs.exe') :
         path.join(pkgPath, 'bin' ,'phantomjs')
 
     try {

--- a/package.json
+++ b/package.json
@@ -1,6 +1,6 @@
 {
   "name": "phantomjs",
-  "version": "2.1.1",
+  "version": "2.1.2",
   "keywords": [
     "phantomjs",
     "headless",

